src/components/Title/__tests__/__snapshots__/text-snapshot-tests.jest.js.snap

```diff
@@ -1,21 +1,21 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Text renders correctly with align 1`] = `
-<span
+<h1
```

Comment:
just making sure this is correct

Reply:
Yes it is a bug fix, bug I will update also the snapshot file name (text => title)